public/themes/custom/hdbt_subtheme/webpack.config.js

```diff
@@ -43,7 +43,6 @@ module.exports = (env, argv) => {
       pathinfo: isDev,
       filename: 'js/[name].min.js',
       publicPath: '../',
-      clean: true,
```

Comment:
Why do you want to remove the `clean: true` setting?

Reply:
We don't. This is a mistake. Fixing.